src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin square($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
```

Comment:
Now this name (square) not correct, coz u create circle in this mixin

Reply:
thanks